jax/_src/numpy/polynomial.py

```diff
@@ -292,6 +293,18 @@ def polymul(a1, a2, *, trim_leading_zeros=False):
   val = convolve(a1, a2, mode='full')
   return val
 
+@_wraps(np.polydiv, lax_description=_LEADING_ZEROS_DOC)
```

Comment:
This does not have a `trim_leading_zeros` argument, so I don't think `_LEADING_ZEROS_DOC` applies.